test/ssc_test/cmod_pvsamv1_test.cpp

```diff
@@ -145,11 +145,13 @@ TEST_F(CMPvsamv1PowerIntegration_cmod_pvsamv1, DefaultResidentialModel)
 
         ssc_number_t lcoe_nom;
         ssc_data_get_number(data, "lcoe_nom", &lcoe_nom);
-        EXPECT_NEAR(lcoe_nom, 7.03, m_error_tolerance_lo) << "Levelized COE (nominal)";
+//        EXPECT_NEAR(lcoe_nom, 7.03, m_error_tolerance_lo) << "Levelized COE (nominal)";
+        EXPECT_NEAR(lcoe_nom, -26.45, m_error_tolerance_lo) << "Levelized COE (nominal)"; // Speedy Gonzales 25yr to 2yr
 
         ssc_number_t lcoe_real;
         ssc_data_get_number(data, "lcoe_real", &lcoe_real);
-        EXPECT_NEAR(lcoe_real, 5.65, m_error_tolerance_lo) << "Levelized COE (real)";
+//        EXPECT_NEAR(lcoe_real, 5.65, m_error_tolerance_lo) << "Levelized COE (real)";
+        EXPECT_NEAR(lcoe_real, -25.50, m_error_tolerance_lo) << "Levelized COE (real)"; // Speedy Gonzales 25yr to 2yr
```

Comment:
Same comment as above

Reply:
lcoe changed from -25.5 to 11.61 with the itc_fed_percent = 0 and itc_sta_percent = 0 in latest commit

npv changed from 5685.3 to -358.2